M2/Macaulay2/tests/normal/trim.m2

```diff
@@ -25,6 +25,13 @@ assert (trim(M,Strategy=>Complement) === trim(N,Strategy=>Complement))
 assert (trim P === R^1)
 assert (trim(P,Strategy=>Complement) === R^1)
 
+-- trim PID
+R=QQ[x]
+I = ideal(3*x^4-x^3+x^2-x-2,3*x^4+2*x^3+6*x+4)
+J = trim I
+assert(I == J)
+assert(numgens J == 1)
```

Comment:
Here's an example that currently fails, if you'd like to use different examples:
```m2
i1 : R = QQ[x]

o1 = R

o1 : PolynomialRing

i2 : I = ideal(x^3-1, x^2+1)

             3       2
o2 = ideal (x  - 1, x  + 1)

o2 : Ideal of R

i3 : trim I
stdio:3:1:(3): error: matrix column index 1 out of range 0 .. 0
```

Reply:
weird, works just fine here.